src/test/java/cristina_savrin/UIStepDefinition.java

```diff
@@ -0,0 +1,143 @@
+package cristina_savrin;
+
+import helpers.Helpers;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import org.apache.commons.lang.RandomStringUtils;
+import org.apache.log4j.Logger;
+import org.hamcrest.Matchers;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.ArrayList;
+
+import static helpers.Helpers.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.assertEquals;
+
+public class UIStepDefinition {
+    WebDriver driver = null;
+    WebDriverWait wait = null;
+    AdoptPetsPage page = null;
+    Logger log = Logger.getLogger(UIStepDefinition.class);
+
+    @After
+    public void after() {
+        driver.close();
+    }
+
+    @Before
+    public void before() {
+
+        var pathToChrome = "src/main/resources/chromedriver.exe";
+        System.setProperty("webdriver.chrome.driver", pathToChrome);
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-debugging-port=9222");
```

Comment:
Can you explin why is used for remote-debugging-port ?

Reply:
I had an issue with ChromeDriver, tried several workarounds from internet and this was the one that helped. 

![image](https://github.com/dgrimaliuc/tekwill-automation-project/assets/135253784/ea6c44fb-15dd-4c02-a6aa-b58179f4e734)

Maybe there is another way..